### PR TITLE
Spotify annotated dataset

### PR DESCRIPTION
**Pull request by ryachen01**

### CODE REVIEW

**Comment by gcampax** [comment]
Also, how far are we in annotating the dialogues you wrote yourself?

**Comment by ryachen01** [comment]
https://github.com/stanford-oval/genie-workdirs/tree/master/mario/spotify/eval

These are the dialogues I manually annotated. Should I do more?

**Comment by gcampax** [comment]
Those look great! Please add them to this PR. We'll collect and annotate data from other sources too, but we can do that incrementally.

File: main/com.spotify/eval/dev/nlu-benchmark/LICENSE [hunk 1]
```diff
@@ -0,0 +1,120 @@
+
+Part of the evaluation set for the Yelp skill is taken from the
```
Comment (gcampax) [inline main/com.spotify/eval/dev/nlu-benchmark/LICENSE:2]: Spotify skill

**Pull request merged**